tests/workspaces/local_workspace_cache_remove_test.py

```diff
@@ -0,0 +1,24 @@
+import pytest
```

Comment:
The test file name does not need to be specific to the one test we are currently running. We can just have a test file for `local_workspace_test.py`. This can be used for adding any other tests about the local workspace too (if needed, later). Similarly for other test files.